lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -0,0 +1,48 @@
+module CC
+  module Engine
+    module Analyzers
+      class EngineConfig
+        attr_reader :config
+
+        def initialize(hash)
+          normalize(hash)
```

Comment:
Thoughts on normalization happening in another class? I don't think this is high priority, but it seems like pulling data out of YAML is a separate concern from loading up configuration.